theano/gpuarray/c_code/dnn_conv_find.c

```diff
@@ -13,7 +13,7 @@ typedef std::unordered_map<std::string, AlgoRec> AlgoCache;
 #endif
 #include "pthread.h"
 
-#line 10 "dnn_conv_find.cc"
+#line 10 "dnn_conv_find.c"
```

Comment:
From what I remarked, this helps to find the right line from where a compilation issue occurs when compilation fails, so it's useful for debugging purposes.

Previously, when a compilation error occurs, g++ returned line numbers based on `mod.cpp`, and then we must infer corresponding line in Theano C code files. Now (with `#line`), g++ directly returns a line number based on Theano C code file (here `dnn_conv_find.c`).